### PR TITLE
test(python): cleanup some unused code

### PR DESCRIPTION
**Pull request by gila**
I had not gotten around to cleaning things up a bit. Ive also removed
the implicit install of Python packages when entering the shell.

### CODE REVIEW

File: test/python/test_nexus.py [hunk 1]
```diff
@@ -236,10 +144,11 @@ async def test_nexus_2_mirror_kill_one(containers, create_nexus):
 
 
 @pytest.mark.asyncio
+@pytest.mark.skip
```
Comment (paulyoong) [inline test/python/test_nexus.py:147]: Is there a reason we are skipping this now?

**Review by paulyoong — APPROVED** [review]

File: shell.nix [hunk 2]
```diff
@@ -76,10 +77,6 @@ in mkShell {
     # SRCDIR is needed by docker-compose files as it requires absolute paths
     export SRCDIR=`pwd`
     # python compiled proto files needed by pytest
```
Comment (tiagolobocastro) [inline shell.nix:79]: move this comment as well?

**Review by tiagolobocastro — APPROVED** [review]

File: test/python/test_nexus.py (hunk 1 above)
Comment (gila) [reply test/python/test_nexus.py:147]: Yeah, the test cant run at the same time/run you need to pick one :) Until we run these things automated it's like, oke, take your pick. working on the automation part.

**Comment by gila** [comment]
bors +r

**Pull request merged**